server.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+
+ const hubsRouter = require('./posts/posts-router');
+
+const server = express();
+
+server.use(express.json());
+server.use('/api/posts', hubsRouter)
+
+server.get('/', (req, res) => {
+    res.send(`
+    <p>Server.js</p>
+    `);
+});
+
+
+module.exports = server;
```

Comment:
Well done Sean! You did really great job, can't wait to see your job on this sprint. I'll recommend you to pay more attention to details and error handling but you clearly have a good grasp of the content. I'm impressed.